build.gradle

```diff
@@ -260,9 +260,8 @@ dependencies {
     compile group: 'uk.gov.hmcts.reform', name: 'http-proxy-spring-boot-autoconfigure', version: '1.1.0'
 
     compile group: 'uk.gov.hmcts.reform', name: 'service-auth-provider-client', version: '3.0.0'
-
+    compile group: 'uk.gov.hmcts.reform', name: 'sscs-common', version: '3.0.66-RC-SSCS-6201-4'
```

Comment:
nothing serious but just to let you know that number ticket in the version is not actually the jira number ticket for this work